bastionzero/service/policies/organizationcontrols.go

```diff
@@ -0,0 +1,141 @@
+package policies
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/bastionzero/bastionzero-sdk-go/bastionzero/service/policies/policytype"
+	"github.com/bastionzero/bastionzero-sdk-go/internal/client"
+)
+
+const (
+	orgControlsBasePath   = policiesBasePath + "/organization-controls"
+	orgControlsSinglePath = orgControlsBasePath + "/%s"
+)
+
+// OrganizationControlsPolicy represents an OrganizationControls policy. This policy controls global mfa.
+type OrganizationControlsPolicy struct {
+	Policy
+
+	mfaEnabled  bool `json:"environments,omitempty"`
```

Comment:
```suggestion
	MFAEnabled  *bool `json:"mfaEnabled,omitempty"`
```

Capitalize the fields, so they're public and can be set--also needed for `json` encoding to work I'm pretty sure. Make them pointer types, so PATCH/`Update` works.